src/explore-education-statistics-admin/src/pages/release/datablocks/ReleaseDataBlockEditPage.module.scss

```diff
@@ -0,0 +1,14 @@
+.dataBlockSelector {
```

Comment:
I think it's possible that using existing components/styles will provide the styles required and/or we should be using the govuk-foo type classnames. I think Marv knows what the GDS toolkit provides and what we could potentially use here.